testgrid/config.yaml

```diff
@@ -5728,12 +5730,15 @@ dashboards:
   dashboard_tab:
   - name: ci-kubernetes-e2e-stable-aws-eks-1-11-prod
     test_group_name: ci-kubernetes-e2e-stable-aws-eks-1-11-prod
-    description: Kubernetes stable branch e2e tests with EKS prod build 1.10
+    description: Kubernetes stable branch e2e tests with EKS prod build 1.11
```

Comment:
uh, are we just testing EKS and not k/k?? I don't see any build or anything like that happening?

Reply:
> are we just testing EKS and not k/k??

Not exactly. Yes we're testing against EKS, but EKS itself is based on the k/k release (which is being updated from time to time). That said, there is still part of the k/k build we're using for the test, like kubectl client code, e2e tests, etc..

Reply:
>  etc..

so just kubectl and the e2e tests? these are not even supported. 1.11 <--> 1.14 is not a supported matrix.

Reply:
Agree, and that's why https://github.com/kubernetes/test-infra/pull/10549#discussion_r244864693